source/task_scheduler.c

```diff
@@ -15,8 +15,37 @@
 
 #include <aws/common/task_scheduler.h>
 
+#include <aws/common/logging.h>
+
+#include <inttypes.h>
+
 static const size_t DEFAULT_QUEUE_SIZE = 7;
 
+const char *aws_task_status_to_c_string(enum aws_task_status status) {
```

Comment:
we already use the term "c_str" in byte_buf.h and string.h
aws_task_status_to_c_str()
or
aws_task_status_c_str()